terraform/app_service.tf

```diff
@@ -53,10 +53,87 @@ resource "azurerm_linux_web_app" "main" {
     mount_path   = "/home/calitp/app/config"
   }
 
+  app_settings = {
+    # app setting used solely for refreshing secrets - see https://github.com/MicrosoftDocs/azure-docs/issues/79855#issuecomment-1265664801
+    "change_me_to_refresh_secrets" = "change me in the portal to refresh all secrets",
+
+    "DOCKER_ENABLE_CI"                    = "true",
+    "DOCKER_REGISTRY_SERVER_URL"          = "https://ghcr.io/",
+    "WEBSITE_HTTPLOGGING_RETENTION_DAYS"  = "99999",
+    "WEBSITE_TIME_ZONE"                   = "America/Los_Angeles",
+    "WEBSITES_ENABLE_APP_SERVICE_STORAGE" = "false",
+    "WEBSITES_PORT"                       = "8000",
+
+    "ANALYTICS_KEY" = local.is_dev ? null : "@Microsoft.KeyVault(VaultName=KV-CDT-PUB-CALITP-${local.env_letter}-001;SecretName=analytics-key)",
+
+    # Django settings
+    "DJANGO_ADMIN"            = (local.is_prod || local.is_test) ? null : "@Microsoft.KeyVault(VaultName=KV-CDT-PUB-CALITP-${local.env_letter}-001;SecretName=django-admin)",
+    "DJANGO_ALLOWED_HOSTS"    = "@Microsoft.KeyVault(VaultName=KV-CDT-PUB-CALITP-${local.env_letter}-001;SecretName=django-allowed-hosts)",
+    "DJANGO_DEBUG"            = local.is_prod ? null : "@Microsoft.KeyVault(VaultName=KV-CDT-PUB-CALITP-${local.env_letter}-001;SecretName=django-debug)",
+    "DJANGO_LOAD_SAMPLE_DATA" = "false",
+    "DJANGO_LOG_LEVEL"        = "DEBUG",
```

Comment:
Can we make this one an env variable too? I imagine we'll want to turn this down in prod.

Reply:
I went ahead and added the values for this to our dev, test, and prod Key Vaults.

Here's the command I ran for dev. I ran similar commands for test and prod.

<img src="https://user-images.githubusercontent.com/25497886/223516777-139d9f8b-249f-4313-9a25-178e217ab5ff.png" width="400">

Reply:
Done in [5cdc6a9](https://github.com/cal-itp/benefits/pull/1294/commits/5cdc6a9b589b13e85153be27f73335eff522848a)

Reply:
Ah, so much cleaner! Diffs will be easier now too I think